risk-map/yaml/risks.yaml

```
@@ -83,14 +83,35 @@ risks:
       exposed:
         - >
           Data Poisoning is exposed during development in the data filtering and processing
-          steps or the training, tuning, and evaluation stages. It’s also exposed
+          steps or the training, tuning, and evaluation stages. It's also exposed
           in the model itself, when it produces inaccurate results, malicious outputs,
           or unexpected behavior.
       mitigated:
         - >
           Proactive mitigation against Data Poisoning happens early in development.
           This includes data sanitization, secure systems and access controls, and
           mechanisms to ensure data and model integrity.
+    mappings:
+      mitre-atlas:
+        - AML.T0020
+        - AML.T0019
+        - AML.T0010.002
```

Should we add `Craft Adversarial Data` https://atlas.mitre.org/techniques/AML.T0043. The whole category is promising or we could get more specific and just use .003 (Manual Modification) and .004 (Insert Backdoor)

Additionally: 
- `Erode Dataset Integrity` https://atlas.mitre.org/techniques/AML.T0059
- `Acquire Infrastructure Domains` https://atlas.mitre.org/techniques/AML.T0008.002

```suggestion
      mitre-atlas:
        - AML.T0020
        - AML.T0019
        - AML.T0010.002
        - AML.T0043
        - AML.T0059
        - AML.T0008.002
```